configure.d/1_queue_limits.conf

```diff
@@ -12,18 +12,18 @@ check() {
 	cur_name=$(basename $2)
 	config_file_path=$1
 
-	if compile_module $cur_name "struct queue_limits q; q.limits_aux;" "linux/blkdev.h"
-	then
-		echo $cur_name "1" >> $config_file_path
-	elif compile_module $cur_name "struct queue_limits q; q.max_write_zeroes_sectors;" "linux/blkdev.h"
+	if compile_module $cur_name "struct queue_limits q; q.max_write_zeroes_sectors;" "linux/blkdev.h"
```

Comment:
Is this refactor actually needed? The only functional change I spotted is adding `exp_q->limits.max_write_zeroes_sectors = 0;` in line 108 (the line number from the original version).
I think Robert refactored this file because he wanted to include here the definition of `cas_queue_max_discard_sectors()` but since it has been extracted to a separate file, maybe the original order would be fine?